draft-ietf-rtcweb-jsep.xml

```diff
@@ -1945,8 +1945,9 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
           <list style="symbols">
 
             <t>The port value is set to the port of the default ICE
-            candidate for this m= section, but given that no candidates
-            have yet been gathered, the "dummy" port value of 9
+            candidate for this m= section, but given that no
+            candidates are gathered and no candidates from the
+            pre-gathered candiate pool are used, the "dummy" port value of 9
```

Comment:
This rationale is getting kind of long. I wonder if we should just say "because JSEP does not included candidates in the initial offer, ..."